kn/fotos/api.py

```diff
@@ -77,12 +84,12 @@ def _set_title(data, request):
         raise PermissionDenied
 
     album.set_title(title)
-    album.save()
+    album.update_visibility([visibility])
```

Comment:
Het is inconsistent als sommige functies wel `.save()` aanroepen en anderen niet.  Als je het op deze manier wilt gebruiken is het misschien beter de functie `update_visibility_and_save` te noemen.